src/components/TransactionBadge.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2018 - 2022 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import styled from 'styled-components'
+
+import Badge from './Badge'
+
+const TransactionBadge = styled(Badge)`
```

Comment:
Is this standalone component really necessary, or should we find a way to reduce differences in Badges design?
This is my bad: I have a hard time deciding between using badges with straight edges (problem: this may be confused with buttons), or 100% round corners...

The former is easier on the eye, as it aligns better in tables etc.

Where I was currently going (tried in the explorer) : rise buttons a bit with a `box-shadow` to imply the possibility to interact; keep badges flat with straight edges, no shadows.

Maybe you have an opinion about this?

I'd propose to take the easy road for now:
- Keep one type of badge only (remove `TransactionBadge`)
- Use straight corners

I'll take care of cleaning this up one a coherent design language emerges.

Reply:
Yes, I like your approach:
- raised (box shadow) elements for buttons
- flat elements for labels/badges

At the moment we are a bit messy. We have 3 complete different components (no inheritance) doing similar jobs:

![image](https://user-images.githubusercontent.com/1579899/155534257-0823a66e-d37b-4b52-af8b-734dd8a9682b.png)

I think that a badge with no background is not a badge at all. That's why I'd propose that:
- Labels are Badges with rounded corners and special coloring. This includes the "NetworkBadge".
- "From" and "Genesis TX" are normal Badges
- "Received" and the amounts are neither Labels nor Badges. The "special" thing about them is that they receive a coloring based on the transaction type. I'd call these "TransactionalInfo".

Reply:
We're on the same page 👌